ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/minio_s3_config.yml

```diff
@@ -0,0 +1,60 @@
+
+###
+# Minio
+# ---
+# Expect the Minio operator to already be setup
+# Provision the Minio instance
+
+- name: Include role 'ocp4-workload-minio'
```

Comment:
Hm. This is unorthodox. Could you run the ocp4-workload-minio before the other workload? Then again that workload is not yet updated to the latest conventions (as evident by the _minio* variables...). So I guess I'm OK with this here.

Reply:
Left as is.